packages/flame/lib/src/experimental/camera_component.dart

```diff
@@ -86,7 +86,10 @@ class CameraComponent extends Component {
   @override
   void renderTree(Canvas canvas) {
     canvas.save();
-    canvas.translate(viewport.position.x, viewport.position.y);
+    canvas.translate(
+      viewport.position.x - viewport.anchor.x * viewport.size.x,
+      viewport.position.y - viewport.anchor.y * viewport.size.y,
+    );
```

Comment:
is this the same as `canvas.translateVector(anchor.translate(viewport.position, viewport.size)`?

we could even have a function to combine an anchor and a viewport that calls `anchor.translate`